buildSrc/src/main/groovy/org/elasticsearch/gradle/SnippetsTask.groovy

```diff
@@ -110,7 +110,8 @@ public class SnippetsTask extends DefaultTask {
                     lastLanguageLine = lineNumber
                     return
                 }
-                if (line ==~ /\/\/ AUTOSENSE\s*/) {
+                if (line ==~ /\/\/ AUTOSENSE\s*/
+                        || line ==~ /\/\/ CONSOLE\s*/) {
```

Comment:
I'd change the regex to:

    /\/\/ *CONSOLE\b/

It's not required to have a space after the `//`